pkg/xen-tools/xen-start

```diff
@@ -46,13 +93,12 @@ xl unpause "$ID" || bail "xl unpause failed"
 # now star polling for domain status (11 sec interval) in the background
 # (note our use of mv to make sure file reads on the other side are atomic)
 (while true; do
-   sleep 11
```

Comment:
Hmm, seems, that we miss sleep here.

Reply:
miss in what sense @giggsoff ?

Reply:
I mean, that now we have the loop with xl info without sleep.